src/pages/clients/statement/Statement.tsx

```diff
@@ -146,6 +163,35 @@ export function Statement() {
     toast.dismiss();
   };
 
+  const handleSendEmail = () => {
+    toast.processing();
+
```

Comment:
```tsx
const send =  client.contacts?.some((contact) => contact.email) || false;

if (!send) {
    return toast.error('client_email_not_set');
}

// ..
```

Reply:
Yeah, agree with you. The logic has been changed!